.github/workflows/merge-autogenerated-content.yml

```
@@ -0,0 +1,45 @@
+name: Merge generated content updates
+
+# Trigger by cronjob
+on:
+  schedule:
+    # Merge the PR every Friday at 10:00 UTC
+    - cron: "0 10 * * FRI"
+  workflow_dispatch: {} # Allow manual triggering
+
+jobs:
+  merge-autogenerated-content:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Generate a token
+      id: generate-token
+      uses: actions/create-github-app-token@v1
```

Let's please pin the version for security

```suggestion
      uses: actions/create-github-app-token@7bfa3a4717ef143a604ee0a99d859b8886a96d00 # v1.9.3
```